robot_names:
  '0': acl_jackal
  '1': acl_jackal2
  '2': sparkal1
  '3': sparkal2
  '4': hathor
  '5': thoth
  '6': apis
  '7': sobek